src/main/java/de/retest/web/WebElementPeer.java

```diff
@@ -121,6 +124,10 @@ private boolean isDefault( final String attributeValue ) {
 		if ( attributeValue.equals( "normal" ) ) {
 			return true;
 		}
+		if ( AttributesProvider.getInstance().getNumericalAttributes().contains( attribute )
```

Comment:
@roesslerj Why do we need the first condition, wouldn't `attributeValue.equals( "0px" )` suffice?

Reply:
Good point, making most of this branch obsolete. Let's discuss that in person...